apps/dashboard/app/models/script.rb

```diff
@@ -193,12 +198,28 @@ def most_recent_job_cluster
     most_recent_job['cluster']
   end
 
+  def create_default_script
+    return if Script.scripts?(project_dir)
+    return if default_script_path.exist?
+
+    script_content = <<~EOF
+      #!/bin/bash
+      # Sample script to configure project defaults. Delete when other scrips available.
```

Comment:
Should be "Delete when other scrip`t`s `are` available."

Reply:
Fixed